README.md

```diff
@@ -44,6 +44,10 @@ npm run start
 Start a chat with `@_discordpuppet_bot:yourserver.com`. When it joins, type
 `help` in the chat to see instructions.
 
+**Warning**: Self-bots are against Discord's Terms of Service.
+* When used with user tokens, you might have your account disabled or terminated.
+* When used with bot tokens, you might lose ability to add new bots or have your account terminated.
```

Comment:
Where do you get this from? this is the first time soru is hearing issues with bot tokens.

Reply:
I asked at DTesters and they said using a bot in a human-like manner is also against ToS. Discord Bot Client is a famous example of what happens when you use bot using a human-targeted client in a human-like manner.

Reply:
And they also said they would do the same even if you did the same thing using a bot developed from scratch.